internal/lb/ping.go

```diff
@@ -30,12 +31,12 @@ func (ps *pingService) Send() (*http.Response, error) {
 		Timestamp: time.Now().Unix(),
 	}
 
-	log.Println("Sending ping to load balancer")
+	log.Info("Sending ping to load balancer")
```

Comment:
verbose?

Reply:
Added log levels so shoule be fine

Reply:
no wait it's not ok, this shouldn't be written on info level every 5 seconds lol.

Should be log.Verbose